ui/Main.php

```diff
@@ -102,14 +107,14 @@ protected function emitNavbar($dispatcher, $action) {
                   "action=".$item['action']."\"><B>".$item['label'] .
                   "</B></A></TD></TR>\n";
         }
-        echo "      <TR><TD COLSPAN=2>&nbsp;</TD></TR>\n";
+        #echo "      <TR><TD COLSPAN=2>&nbsp;</TD></TR>\n";
         if($this->session->isAuth("u")) {
-            echo "      <TR><TD></TD><TH CLASS=\"nav3s\">" . $this->session->getDN() . " is logged in</TH></TR>\n";
+            #echo "      <TR><TD></TD><TH CLASS=\"nav3s\">" . $this->session->getDN() . " is logged in</TH></TR>\n";
```

Comment:
Restored blank link and user name (now placed below the Logout link). Kept the 'L' though because the spacing and coloring provide sufficient distinction between this link and the others and because it is important to use consistent capitalization.